tests/resources/test_bills.py

```diff
@@ -59,6 +60,21 @@ def test_cancel_bill(client):
     assert updated_transaction.status == 'refunded'
 
 
+@pytest.mark.vcr
+def test_cancel_bill_fail(client):
+    transactions = client.transactions.list()
+    transactions = list(filter(lambda t: t.status == 'fulfilled',
```

Comment:
We stay away from lambdas, filters, and other functional programming functions in Python—unless there's no cleaner way to do it. In this case, a list comprehension with an `if` clause would be more readable and consistent with the rest of our code.